src/examples/axis.jsx

```diff
@@ -0,0 +1,95 @@
+// @flow
+import React, { Component } from 'react';
+import JubilationChart from '../jubilation-chart';
+import JubilationScatter from '../jubilation-scatter';
+import JubilationTheme from '../jubilation-theme';
+import { XAxis, YAxis } from '../jubilation-axis';
+
+function randomPoints() {
+  return [1,2,3,4,5,6,7,8,9,10].map(() => ({ x: Math.random() * 600, y: Math.random() * 300 }))
+}
+
+class AxisExample extends Component {
+  state = {
+    points: randomPoints(),
+    colors: [0, 1],
+  }
+
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.updateNumX = this.updateNumX.bind(this);
+    this.updateNumY = this.updateNumY.bind(this);
+    this.state = {
+      showX: true,
+      numXTicks: 2,
+      extendXTicks: true,
+      showY: true,
+      numYTicks: 2,
+      extendYTicks: true,
+      points: randomPoints(),
+    };
+  }
+
+  toggle(axis) {
+    this.setState({
+      [axis]: !this.state[axis]
+    })
+  }
+
+  updateNumX(event) {
+    this.setState({
+      numXTicks: event.target.value
+    })
+  }
+
+  updateNumY(event) {
+    this.setState({
+      numYTicks: event.target.value
+    })
+  }
+
+  render() {
+    const { showX, numXTicks, extendXTicks, showY, numYTicks, extendYTicks } = this.state;
+
+    return <div>
+      <form>
```

Comment:
yeah, you could write a little function that takes a label text, input type and change handler to spit out all of the repeated label and inputs here but honestly it would end up being about the same amount of code and would probably be less readable. I'm cool to just leave it as is